app/TDG/RoomTDG.py

```diff
@@ -22,20 +22,14 @@ def findAll():
     else:
         return []
 
-def insert(lock):
-    conn = connect_db()
-    if conn:
-	cur = conn.cursor()
-	cur.execute("""INSERT INTO roomTable(lock) VALUES (%s);""", lock)
-	conn.commit()
-	conn.close()
 
-def update(id, availability):
+
+def update(id):
```

Comment:
Now that rooms don't have any data other than IDs, updates don't make much sense. You can remove this function altogether